### DISCUSSION TITLE
Trouble getting library from an external drive with docker

### DISCUSSION THREAD

**Discussion by JangoFett007** [advplyr/audiobookshelf#883] [Q&A]
This is my first time using docker, so I could be missing something very obvious.

I have my audio books on a second HDD, but can't seem to get audiobookshelf to recognize it.
I copied a few books to the folder on my c drive where docker installed audiobookshelf and it works with those, I just can't get anything outside of that folder to work.

Any help would be very appreciated!

**Comment by advplyr** [comment]
There are many experienced users in the discord server that help people get setup all the time. I think that's the best option because you'll need to share your current setup and all that.
